### PR TITLE
OSDOCS-4115: adds Hawkular migration limitation to monitoring conside…

### PR DESCRIPTION
**Pull request by JoeAldinger**
[OSDOCS-4115](https://issues.redhat.com//browse/OSDOCS-4115)-4.11: adds limitation statement to 4.11 documentation

Version(s):
4.11+

Issue:
[Jira](https://issues.redhat.com/browse/OSDOCS-4115)

Link to docs preview:
Direct [link](https://51448--docspreview.netlify.app/openshift-enterprise/latest/migrating_from_ocp_3_to_4/planning-migration-3-4.html)

QE review:
- [x] QE has approved this change.

Additional information:
This is related to the conversation around GH #51061 and will require change management.

### CODE REVIEW

**Comment by mffiedler** [comment]
LGTM

**Comment by djfjeff** [comment]
LGTM

**Review by tmalove — COMMENTED** [review]
/remove-label peer-review-needed
/label peer-review-in-progress

File: migrating_from_ocp_3_to_4/planning-migration-3-4.adoc [hunk 1]
```diff
@@ -211,7 +211,7 @@ For more information, see xref:../authentication/managing-security-context-const
 [id="migration-preparing-monitoring"]
 === Monitoring considerations
 
-Review the following monitoring changes to consider when transitioning from {product-title} 3.11 to {product-title} {product-version}.
+Review the following monitoring changes to consider when transitioning from {product-title} 3.11 to {product-title} {product-version}. You cannot migrate Hawkular configs and metrics to Prometheus.
```
Comment (tmalove) [inline migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:214]:
```suggestion
Review the following monitoring changes to consider when transitioning from {product-title} 3.11 to {product-title} {product-version}. You cannot migrate Hawkular configurations and metrics to Prometheus.
```

Comment (tmalove) [reply migrating_from_ocp_3_to_4/planning-migration-3-4.adoc:214]: Here is an additional suggestion for the same sentence...
```suggestion
Review the following monitoring changes when transitioning from {product-title} 3.11 to {product-title} {product-version}. You cannot migrate Hawkular configs and metrics to Prometheus.
```

**Review by tmalove — COMMENTED** [review]
/remove-label peer-review-in-progress
/label peer-review-done

A couple of suggestions...thx.

**Comment by rbaumgar** [comment]
Hawkular was already deprecated in OpenShift 3.11. See https://docs.openshift.com/container-platform/3.11/release_notes/ocp_3_11_release_notes.html#ocp-311-major-changes-in-40

**Comment by JoeAldinger** [comment]
@rbaumgar for our customers' clarity, we wanted to document the metrics in both 3.11 and 4.11.

**Review by maxwelldb — APPROVED** [review]

**Pull request merged**